### ISSUE TITLE
compatibility issues with laravel 8

### ISSUE THREAD

**Issue by C2H6-383** [felixdorn/laravel-rickroll#7]
I am currently trying to upgrade to laravel 8. So I updated my composer.json file to the new version. Composer gave me the following error when running `composer update`:

```
Loading composer repositories with package information                                                                                                                                                                                         
Updating dependencies (including require-dev)                                                                                                                                                                                                  
Your requirements could not be resolved to an installable set of packages.                                                                                                                                                                     
                                                                                                                                                                                                                                               
  Problem 1                                                                                                                                                                                                                                    
    - Conclusion: don't install laravel/framework v8.11.2                                                                                                                                                                                      
    - Conclusion: don't install laravel/framework v8.11.1                                                                                                                                                                                      
    - Conclusion: don't install laravel/framework v8.11.0                                                                                                                                                                                      
    - Conclusion: don't install laravel/framework v8.10.0                                                                                                                                                                                      
    - Conclusion: don't install laravel/framework v8.9.0                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.8.0                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.7.1                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.7.0                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.6.0                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.5.0                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.4.0                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.3.0                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.2.0                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.1.0                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.0.4                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.0.3                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.0.2                                                                                                                                                                                       
    - Installation request for felixdorn/laravel-rickroll ^1.0 -> satisfiable by felixdorn/laravel-rickroll[1.0.0].                                                                                                                            
    - Conclusion: don't install laravel/framework v8.0.1                                                                                                                                                                                       
    - Conclusion: don't install laravel/framework v8.0.0                                                                                                                                                                                       
    - felixdorn/laravel-rickroll 1.0.0 requires illuminate/support ^7 -> satisfiable by laravel/framework[v7.28.4, 7.x-dev], illuminate/support[7.x-dev, v7.0.0, v7.0.1, v7.0.2, v7.0.3, v7.0.4, v7.0.5, v7.0.6, v7.0.7, v7.0.8, v7.1.0, v7.1.1
, v7.1.2, v7.1.3, v7.10.0, v7.10.1, v7.10.2, v7.10.3, v7.11.0, v7.12.0, v7.13.0, v7.14.0, v7.14.1, v7.15.0, v7.16.0, v7.16.1, v7.17.0, v7.17.1, v7.17.2, v7.18.0, v7.19.0, v7.19.1, v7.2.0, v7.2.1, v7.2.2, v7.20.0, v7.21.0, v7.22.0, v7.22.1,
 v7.22.2, v7.22.3, v7.22.4, v7.23.0, v7.23.1, v7.23.2, v7.24.0, v7.25.0, v7.26.0, v7.26.1, v7.27.0, v7.28.0, v7.28.1, v7.28.2, v7.28.3, v7.28.4, v7.3.0, v7.4.0, v7.5.0, v7.5.1, v7.5.2, v7.6.0, v7.6.1, v7.6.2, v7.7.0, v7.7.1, v7.8.0, v7.8.1
, v7.9.0, v7.9.1, v7.9.2].                                                                                                                                                                                                                     
```

I am using Windows 10, PHP 7.4.0, Composer 1.10.16

I have rerun this command multiple times, but the error was always the same.

**Comment by felixdorn** [comment]
I'll make it compatible with L8 this week, sorry for the delay.

**Comment by C2H6-383** [comment]
no problem, thank you very much

**Comment by felixdorn** [comment]
Closed, see #8
